packages/mainmenu-extension/src/index.ts

```diff
@@ -342,14 +347,17 @@ export function createFileMenu(
     execute: Private.delegateExecute(app, menu.consoleCreators, 'createConsole')
   });
 
-  commands.addCommand(CommandIDs.quit, {
-    label: 'Quit',
-    caption: 'Quit JupyterLab',
+  commands.addCommand(CommandIDs.shutdown, {
+    label: 'Shutdown',
+    caption: 'Shut down JupyterLab',
     execute: () => {
       return showDialog({
-        title: 'Quit confirmation',
-        body: 'Please confirm you want to quit JupyterLab.',
-        buttons: [Dialog.cancelButton(), Dialog.warnButton({ label: 'Quit' })]
+        title: 'Shut down confirmation',
```

Comment:
I looked it up, and apparently it's the *noun* that is used as an adjective, so I guess this should be "Shutdown confirmation". Sorry, apparently I'm still learning English here...

Reply:
This is taken care of